pkg/cmd/migrate.go

```diff
@@ -29,6 +29,14 @@ func NewMigrateCommand() *cobra.Command {
 		RunE:  runMigration,
 	}
 
+	viper.SetConfigName("config")
```

Comment:
I'd prefer we do this in way that shares this logic.  For example, something like

```
package config

func SetupConfig(v *viper.Viper) {
    ...
}
```
and then use `SetupConfig` in the `run` and the `migrate` CLI targets.

Reply:
I found the function you linked, but potential problem I saw with using that function is that it returns a specific `Config` struct, and I wasn't sure if the two commands should deal with the exact same config, or to consider the configurations merely as overlapping.
In other words: could it be confusing that in `run` (and potentially other commands in the future), the config has an option `migrateToVersionAndStop` or `DoAllMigrationsInOneBigTransaction`, because it is needed in the `migrate` command